pyupgrade.py

```diff
@@ -1084,6 +1085,27 @@ def visit_Call(self, node):  # type: (ast.Call) -> None
                 not _starargs(node)
         ):
             self.native_literals.add(_ast_to_offset(node))
+        elif (
+                isinstance(node.func, ast.Attribute) and
+                isinstance(node.func.value, ast.Str) and
+                node.func.attr == 'encode' and
+                not _starargs(node) and
+                len(node.args) == 1 and
+                isinstance(node.args[0], ast.Str) and
+                node.args[0].s.lower() == 'utf-8'
```

Comment:
`'utf8'` works too -- there's a function in either `codecs` or `encodings` that normalizes this value -- can probably reuse that here?

Reply:
Sure

Reply:
Hmm I found this one but it seems less than useful

```
In [5]: encodings.normalize_encoding('utf-8')
Out[5]: 'utf_8'

In [6]: encodings.normalize_encoding('utf8')
Out[6]: 'utf8'

In [7]: encodings.normalize_encoding('UTF8')
Out[7]: 'UTF8'
```

I wasn't able to find anything in codecs (from ipython autocomplete). Any advice?

Reply:
```pycon
>>> codecs.lookup('utf_8').name
'utf-8'
>>> codecs.lookup('utf8').name
'utf-8'
>>> codecs.lookup('UTF8').name
'utf-8'
```

Reply:
ty!